app/views/admin/_admin_nav.html.erb

```diff
@@ -1,15 +1,14 @@
-<h3 class="landmark heading"><%= ts("Admin Navigation") %></h3>
+<h3 class="landmark heading"><%= t(".landmark") %></h3>
 <ul class="navigation actions" role="navigation">
   <li>
-    <%= span_if_current ts("AO3 News", key: "header"), admin_posts_path %>
+    <%= span_if_current t(".ao3_news"), admin_posts_path %>
    </li>
   <li>
-    <%= span_if_current ts("Post AO3 News", key: "header"),
-                        new_admin_post_path %>
-</li>
+    <%= span_if_current t(".post_ao3_news"), new_admin_post_path %>
+  </li>
   <% if params[:controller] == "admin_posts" && params[:action] == "edit" %>
     <li>
-      <%= link_to t("admin.admin_nav.delete", default: "Delete Post"),
```

Comment:
This key is mentioned in `ignore_missing` in i18n-tasks.yml, it would be nice if you removed this file from the comment there.